airbyte-integrations/bases/standard-destination-test/src/main/resources/v0/edge_case_catalog.json

```diff
@@ -67,16 +67,6 @@
         }
       }
     },
-    {
-      "name": "reserved_keywords",
-      "json_schema": {
-        "properties": {
-          "order": {
-            "type": "string"
-          }
-        }
-      }
-    },
```

Comment:
Galaxy doesn't allow column name with reserved keywords. Please advise the correct way to skip this test. Should I override the `DestinationAcceptanceTest#testSync` to ignore specific test?

Reply:
You must override the test and added a meaningful error to users. Besides that  you must add a WARN note in the documentation linking to Starburst Galaxy reserved words.

Reply:
It's not required anymore as now as connector supports reserved keywords now.